docs/framework/tools/developer-command-prompt-for-vs.md

```diff
@@ -14,17 +14,39 @@ ms.assetid: 94fcf524-9045-4993-bfb2-e2d8bad44219
 ---
 # Developer Command Prompt for Visual Studio
 
-Developer Command Prompt for Visual Studio enables you to use .NET Framework tools more easily. It's a command prompt that automatically sets specific environment variables. After opening Developer Command Prompt, you can enter the commands for [.NET Framework tools](index.md) such as `ildasm` or `clrver`.
+Developer Command Prompt for Visual Studio is a command prompt that has specific environment variables set. It enables you to use command-line developer tools more easily. After opening Developer Command Prompt for Visual Studio, you can enter the commands for different utilities without having to know where they're located. The commands you can run from Developer Command Prompt include:
+
+- [`MSBuild`](/visualstudio/msbuild/msbuild-command-line-reference), to build a project or solution.
+- [.NET CLI commands](../../core/tools/index.md), such as [`dotnet`](../../core/tools/dotnet.md) and [`dotnet run`](../../core/tools/dotnet-run.md).
```

Comment:
.NET CLI commands should normally be available outside developer command prompt.